app/views/users/edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1><%= t(".title", name: user.name) %></h1>
+<%= simple_form_for user do |f| %>
+  <%= f.input :name %>
+  <%= f.input :email %>
+  <%= f.submit t(".submit") %>
```

Comment:
Check `simple_form` documentation about how to get the translation to load automatically for the submit button

Reply:
Done with sha: 5ee2ab245060a9ca7a6a25f2e6e1064039738a77

Reply:
Done with sha: 4330bb5e0a56fc61dc659cf105600219ff954435